packages/specs/protocol/data-structures.md

```diff
@@ -0,0 +1,348 @@
+# Optimistic Ethereum Data Structures
```

Comment:
Feel free to leave a comment here where you think a variable name should change, but IMO actually updating it should be reserved for a new PR.